frontend/src/App.js

```diff
@@ -4,10 +4,11 @@ import { useSelector } from "react-redux";
 import NavBar from "./components/NavBar";
 import Carousel from "./components/Carousel";
 import SignIn from "./components/SignIn";
-import Form from "./components/Form";
 import NewsFeed from "./components/NewsFeed";
+import UserPage from "./components/UserPage";
 import ContainerEditProfile from "./components/EditProfile/ContainerEditProfile";
 
+
```

Comment:
(This second empty lines makes me think your Prettier is misconfigured and will wreck havoc in the project)